discovery_engine/lib/src/domain/feed_manager.dart

```diff
@@ -131,4 +138,34 @@ class FeedManager {
 
     return const EngineEvent.clientEventSucceeded();
   }
+
+  /// Adds a source to excluded sources set.
+  Future<EngineEvent> addExcludedSource(String source) async {
+    if (source.isEmpty) {
+      throw ArgumentError('source "$source" can\'t be empty');
```

Comment:
```suggestion
      throw ArgumentError('source can\'t be empty');
```
since `source` is empty we can remove `$source` here